Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -87,7 +88,7 @@ public static void SetPlaceHolder(DependencyObject obj, string value)
         /// </summary>
         /// <param name="obj">TextBox control</param>
         /// <returns>CustomMask value</returns>
-        public static string GetCustomMask(DependencyObject obj)
+        public static string GetCustomMask(TextBox obj)
```

Comment:
Can you change the sample to ensure that we have use both x:Bind and Binding.. I have a feeling TextBox is going to bust x:Bind